client/source/DelphiLint.ToolFrame.dfm

```diff
@@ -264,10 +328,292 @@ object LintToolFrame: TLintToolFrame
     object ActionClearActiveFile1: TMenuItem
       Action = PluginCore.ActionClearActiveFile
     end
+    object Separator2: TMenuItem
+      Caption = '-'
+    end
+    object ViewLogItem: TMenuItem
+      Caption = '&View Last Analysis Log'
+      OnClick = ErrorButtonClick
```

Comment:
This event handler name references the error button but is also used by the `WarningButton` and `ViewLogItem`. Should the name be something else?

Reply:
Changed to `ViewLogClick`.